### PR TITLE
Add size-limit action

### PR DESCRIPTION
**Pull request by gregone**
~~Uses the [size-limit](https://github.com/marketplace/actions/size-limit-action) action to report asset sizes.~~

Now uses ember-asset-size instead

### CODE REVIEW

**Review by sabrinako — APPROVED** [review]
Nice! LGTM but I haven't worked with GH actions much so I'd wait for @jgwhite's approval too

**Comment by jgwhite** [comment]
@sabrinako don’t sell yourself short. Your approval is more than enough!

**Review by jgwhite — APPROVED** [review]
This rules 🥳

**Pull request merged**
